pkg/webhook/hook.go

```diff
@@ -129,11 +130,13 @@ func (whsvr *webhookServer) mutate(ar *v1beta1.AdmissionReview) *v1beta1.Admissi
 
 	clone := pod.DeepCopy()
 	whsvr.trySetNodeName(clone, req.Namespace)
-	inject(clone, whsvr.ignoreSelectorKeys)
+	if err := inject(clone, whsvr.ignoreSelectorKeys); err != nil {
```

Comment:
If we add error, please use it. What more, the `err` declared in previous has been would be override